src/Misc/layoutbin/en-US/strings.json

```diff
@@ -249,6 +249,8 @@
   "DownloadArtifacts": "Download Artifacts",
   "DownloadArtifactsFailed": "Downloading artifacts failed: {0}",
   "DownloadArtifactTo": "Download artifact to: {0}",
+  "DownloadBuildArtifactWarning": "Please use Download Build Artifact task for downloading 'Build Artifact' type artifact. https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-build-artifacts?view=azure-devops",
```

Comment:
These need to be in alphabetical order or a unit test fails. Can you change this to "DownloadArtifactWarning' and instead of 'Build Artifact' or 'UNC', put '{0}' and then just pass in 'Build Artifact' or 'UNC' when you call it.

```
context.Warning(StringUtil.Loc("DownloadArtifactWarning", "UNC"));
```

Reply:
Done.